### ISSUE TITLE
Build errors

### ISSUE THREAD

**Issue by krazyjakee** [mikesmullin/node-gd#10]
Been at this for a couple hours so I hope you can help me.

```
node-gd.cpp
..\cpp\node-gd.cpp(17): fatal error C1083: Cannot open include file: 'gd.h': No such file or directory [node_modules\node-gd\build\node_gd.vcxproj]
```

**Comment by furagu** [comment]
Hey there! Can we please have this pushed to NPM?

**Comment by y-a-v-a** [comment]
Build paths for Brew and Macports are available now on osx since v0.3.0
